src/EnergyPlus/General.cc

```diff
@@ -1255,26 +1255,12 @@ Real64 BlindBeamBeamTrans(Real64 const ProfAng,        // Solar profile angle (r
 }
 
 Real64 POLYF(Real64 const X,         // Cosine of angle of incidence
```

Comment:
Given that in a number of callsites `X` is 1, I wonder if it makes sense to make this function `constexpr`.